enocean/eep_parser.py

```diff
@@ -181,21 +184,28 @@ def _parse_eep_A5_38_08(self, payload, status):
         return results
 
     def _parse_eep_A5_3F_7F(self, payload, status):
-        #logger.debug("enocean: processing A5_3F_7F")
+        logger.debug("enocean: processing A5_3F_7F")
         results = {'DI_3': (payload[3] & 1 << 3) == 1 << 3, 'DI_2': (payload[3] & 1 << 2) == 1 << 2, 'DI_1': (payload[3] & 1 << 1) == 1 << 1, 'DI_0': (payload[3] & 1 << 0) == 1 << 0}
         results['AD_0'] = (((payload[1] & 0x03) << 8) + payload[2]) * 1.8 / pow(2, 10)
         results['AD_1'] = (payload[1] >> 2) * 1.8 / pow(2, 6)
         results['AD_2'] = payload[0] * 1.8 / pow(2, 8)
         return results
 
+
+####################################################
+### --- Definitions for RORG = D5 / ORG = 06 --- ###
+####################################################
     def _parse_eep_D5_00_01(self, payload, status):
-        #ORG = 0x06
-        #Window/Door Contact Sensor, for example Eltako FTK, FTKB
-        #logger.debug("enocean: processing D5_00_01: Door contact")
+        # Window/Door Contact Sensor, for example Eltako FTK, FTKB
+        logger.debug("enocean: processing D5_00_01: Door contact")
```

Comment:
Comment out, as mentioned before. This applies to all debug logs the you uncommented.